src/components/BottomBar/LocationPicker.vue

```diff
@@ -28,6 +36,32 @@ export default class LocationPicker extends Vue {
     this.$store.commit(SET_CITY, { city });
   }
 
+  async getCurrentCity() {
+    await geolocationService.getCurrentPosition().then(async (coords) => {
+      this.$store.commit(SET_LAT, coords.latitude);
+      this.$store.commit(SET_LON, coords.longitude);
+      // This might not be the best approach to displaying the City name and
+      // the converstion from coords to city might be alittle off
+      // TODO: We could remove the city setting all together and blank out
+      // the city since they are searching by coordinates and may not be in a city
+      const location: GeoDirectResponse[] = await openWeatherService
+        .searchCityByCoords(coords.latitude, coords.longitude);
+      this.$store.commit(SET_CITY, { city: location[0].name });
+
+      const weather: OneCallWeather = await openWeatherService
+        .getOneCallWeather(coords.latitude, coords.longitude);
+      this.$store.commit(SET_WEATHER, weather);
+    }).catch((error) => {
+      this.$bvToast.toast(`I'm sorry, there seems to be an issue. Error Message: ${error.message}`,
+        {
+          title: 'Oopsy-Daisy',
+          variant: 'danger',
+          solid: true,
+          toaster: 'b-toaster-top-center',
+        });
```

Comment:
I tried to think of a Minnesota error message, maybe that'll add to our "secret sauce"

Reply:
Haha, I love it! BTW, somehow I missed the notification that this was ready for review. Sorry it sat so long! I'll take a look at the rest of it this weekend.